packages/models/src/lib/persist-config.ts

```diff
@@ -5,7 +5,7 @@ export const formatSchema = z.enum(['json', 'md']);
 export type Format = z.infer<typeof formatSchema>;
 
 export const persistConfigSchema = z.object({
-  outputDir: filePathSchema('Artifacts folder'),
+  outputDir: filePathSchema('Artifacts folder').default('.code-pushup'),
```

Comment:
I believe it would be better to add the default value directly to `yargs` which would also provide this information in the `help` command, making it more accessible for the users.

Reply:
done